styles/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+    background-color:beige;
+}
+
+
+h1 {
+    font-family:'Franklin Gothic Medium';
+    font-size:xx-large;
+    color: rgb(152, 175, 118);
+    text-align: center;
+    
+}
+
+
+main {
+    text-align: center;
+    font-family: 'Franklin Gothic Medium';
+}
+
+
+[href="#"]:hover {
+    text-decoration: underline;
+    color: orange;
+    
+}
+
+[href="#about"]:hover {
+    text-decoration: underline;
+    color: orange;
+}
+
+[href="#contact"]:hover {
+    text-decoration: underline;
+    color:orange;
+}
+
+[href="pages/portfolio.html"]:hover {
+    text-decoration: underline;
+    color:orange;
+}
+
+[href="pages/about.html"]:hover {
+    text-decoration: underline;
+    color:orange;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
Initially, I didn't think this `flex` had an observable effect since the container div only has one child, but it ends up affecting the vertical alignment with the `align-content`. This leads to the header nav looking a little different on the non-index page (since they don't have this style rule), which given the change in border color was maybe intentional?